vite-nohttps.config.js

```diff
@@ -0,0 +1,12 @@
+import { sveltekit } from '@sveltejs/kit/vite';
```

Comment:
This is just for Stackblitz, right? Maybe we should add a comment at the top for people to know that if they clone this, working locally, they won't need it?

Reply:
added the comment on the file